examples/voice-pipeline-agent/minimal_assistant.py

```diff
@@ -48,7 +48,7 @@ async def entrypoint(ctx: JobContext):
         vad=ctx.proc.userdata["vad"],
         stt=deepgram.STT(model=dg_model),
         llm=openai.LLM(),
-        tts=openai.TTS(),
```

Comment:
Let's not change the minimal_assistant code

Reply:
will not be merged.. this is for the Play.ai team to test